### PR TITLE
adding documentation for ur and ur'

### PR DESCRIPTION
**Pull request by sfradkin**

### CODE REVIEW

**Comment by yaxu** [comment]
I'm nervous about this because it's a work in process.  How about we delete ur, rename ur' to ur, and then put in the documentation with a warning that the interface might change?

File: _functions/compositions/ur.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ur' :: Time -> Pattern String -> [(String, Pattern a)] -> [(String, Pattern a -> Pattern a)] -> Pattern a
+~~~~
+
+`ur'` allows for the meta-composition of patterns. The first argument is a number of cycles
+that the `ur'` function loops at. The function will automatically split up the patterns defined
+across that number of cycles. The second argument is a pattern that defines how to compose
+the patterns to be defined. The third argument is a named array of Tidal patterns. The fourth argument is a named array of transition functions.
+
+~~~~ haskell
+d1 $ ur' 4 "[bd1 sn1]"
```
Comment (willbasky) [inline _functions/compositions/ur.md:20]: May be, do you mean `ur`

Comment (sfradkin) [reply _functions/compositions/ur.md:20]: There is a function called `ur`, but we will be removing that function in an upcoming release.  We are favoring the usage of `ur'`.

Comment (willbasky) [reply _functions/compositions/ur.md:20]: So, `ur'` will be for both:
``` haskell
ur :: Time -> Pattern String -> [Pattern a] -> Pattern a
```
and for
``` haskell
ur' :: Time
     -> Pattern String
     -> [(String, Pattern a)]
     -> [(String, Pattern a -> Pattern a)]
     -> Pattern a
```
??

Comment (sfradkin) [reply _functions/compositions/ur.md:20]: No.  The function definition for `ur` will not be available anymore.  `ur'` does everything that `ur` does plus more.

**Pull request closed without merging**